modules/test/playwright/tests/frontend-taglib/pages/SamplePage.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+import {ApiHelpers} from '../../../helpers/ApiHelpers';
+import {liferayConfig} from '../../../liferay.config';
+import getRandomString from '../../../utils/getRandomString';
+import getPageDefinition from '../../layout-content-page-editor-web/utils/getPageDefinition';
+import getWidgetDefinition from '../../layout-content-page-editor-web/utils/getWidgetDefinition';
+
+export class SamplePage {
+	readonly apiHelpers: ApiHelpers;
+	readonly page: Page;
+	readonly linkList: Locator;
+
+	constructor(page: Page) {
+		this.apiHelpers = new ApiHelpers(page);
+		this.page = page;
+		this.linkList = page.getByRole('link');
+	}
+
+	async selectLink(tabName: string) {
+		const linkHeading = this.linkList.getByText(tabName);
+
+		await expect(linkHeading).toBeInViewport();
+
+		await linkHeading.click();
+	}
+
+	async setupSampleWidget({site}) {
+		const widgetDefinition = getWidgetDefinition({
+			id: getRandomString(),
+			widgetName: 'com_liferay_sample_web_portlet_SamplePortlet',
```

Comment:
The portlet name here looks wrong. Shouldn't it match portlet Java package? This is not wrong in PR, I think this is wrong in master, not only for `frontent-taglib-sample-web`, but also for `frontent-taglib-clay-sample-web`

Reply:
It makes sense. I guess they should be:
_com_liferay_frontend_taglib_sample_web_portlet_SamplePortlet
com_liferay_frontend_taglib_clay_sample_web_portlet_ClaySamplePortlet_

@dsanz Does this make sense to you?